Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Menu/MenuPage.xaml

```diff
@@ -0,0 +1,414 @@
+                                </VisualStateGroup>
+                                <VisualStateGroup x:Name="CheckPlaceholderStates">
+                                    <VisualState x:Name="NoPlaceholder" />
+                                    <VisualState x:Name="CheckPlaceholder">
+                                        <Storyboard>
+                                            <ObjectAnimationUsingKeyFrames Storyboard.TargetName="TextBlock"
+                                                                           Storyboard.TargetProperty="Margin">
+                                                <DiscreteObjectKeyFrame KeyTime="0"
+                                                                        Value="{ThemeResource MenuFlyoutItemPlaceholderThemeThickness}" />
+                                            </ObjectAnimationUsingKeyFrames>
+                                        </Storyboard>
+                                    </VisualState>
+                                </VisualStateGroup>
+                                <VisualStateGroup x:Name="PaddingSizeStates">
+                                    <VisualState x:Name="DefaultPadding" />
+                                    <VisualState x:Name="NarrowPadding">
+                                        <Storyboard>
+                                            <ObjectAnimationUsingKeyFrames Storyboard.TargetName="LayoutRoot"
+                                                                           Storyboard.TargetProperty="Padding">
+                                                <DiscreteObjectKeyFrame KeyTime="0"
+                                                                        Value="{ThemeResource MenuFlyoutItemThemePaddingNarrow}" />
+                                            </ObjectAnimationUsingKeyFrames>
+                                        </Storyboard>
+                                    </VisualState>
+                                </VisualStateGroup>
+                            </VisualStateManager.VisualStateGroups>
+                        </Grid>
+                    </ControlTemplate>
+                </Setter.Value>
+            </Setter>
+        </Style>
+    </Page.Resources>
+
+    <Grid Background="{StaticResource Brush-Grey-05}">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto" />
+            <RowDefinition />
+        </Grid.RowDefinitions>
+
+        <controls:Menu MenuFlyoutStyle="{StaticResource MenuStyle}"
+                       Orientation="Horizontal">
+            <controls:MenuItem controls:Menu.InputGestureText="Shift+F"
+                               Header="File">
+                <MenuFlyoutSubItem Text="New">
+                    <MenuFlyoutItem controls:Menu.InputGestureText="Ctrl+Shift+N"
+                                    Command="{StaticResource NewProject}"
+                                    Style="{StaticResource MenuFlyoutItemStyle}"
```

Comment:
As I told you before for a reason or another implicit style is not working for MenuFlyoutItem inside MenuFlyoutSubItem